HeinOnline.js

```diff
@@ -0,0 +1,158 @@
+{
+	"translatorID": "3dcbb947-f7e3-4bbd-a4e5-717f3701d624",
+	"label": "HeinOnline",
+	"creator": "Frank Bennett",
+	"target": "^https?://heinonline\.org/HOL/(?:LuceneSearch|Page)\\?",
```

Comment:
Double backslashes, otherwise you're just escaping the period for the string, which is still an unescaped period in RegExp. Use single if you're typing in Scaffold.